fleet/lib/htcondor/overlays/manke/htcondor-central-manager-service.yaml

```diff
@@ -9,7 +9,6 @@ metadata:
     name: htcondor-central-manager
     visualize: "true"
```

Comment:
What is the purpose of this label?

Reply:
was removed, don't know where I got it from... silly me.

Reply:
removed